src/admin-views/aggregator/settings.php

```diff
@@ -408,7 +408,7 @@
 		'tribe_aggregator_default_url_import_event_settings' => array(
 			'type' => 'dropdown',
 			'label' => esc_html__( 'Import Event Settings', 'the-events-calendar' ),
-			'tooltip' => esc_html__( "Fetch source event's settings (e.g. Show Google Maps Link or Sticky in Month View) when importing from another site using The Events Calendar.", 'the-events-calendar' ),
+			'tooltip' => esc_html__( 'Fetch source events settings (e.g. Show Google Maps Link or Sticky in Month View) when importing from another site using The Events Calendar.', 'the-events-calendar' ),
```

Comment:
I checked with our team grammarian and we feel the apostrophe is appropriate here, given the context, so rolling back this change would be ideal.

Reply:
...but keep the other change

Reply:
ready for another review